app/helpers/vm_helper/textual_summary.rb

```diff
@@ -333,7 +333,7 @@ def textual_flavor
 
   def textual_service
     h = {:label => _("Service"), :image => "service"}
-    service = @record.service
+    service = @record.service || (@record.respond_to?(:orchestration_stack) && @record.orchestration_stack.try(:service))
```

Comment:
do we know why `@record.service` does not already have the service?

may simplify to `@record.try(:orchestration_stack).try(:service)`